src/components/shortcuts.js

```diff
@@ -200,12 +213,12 @@ function executeAction(card, target, action) {
         });
     } else if (action === 'play' || action === 'resume') {
         const startPositionTicks = parseInt(card.getAttribute('data-positionticks') || '0', 10);
-
```

Comment:
I would keep this empty line, so the code doesn't look too tight. And less unnecessary changes.

Reply:
I think lint got mad at me earlier by keeping this as a newline?

Maybe it was because there was a few tabs in them? Will also test later today!

Thank you :)